src/index.ts

```diff
@@ -107,11 +108,33 @@ export default runExtension(async (args) => {
 
     toggleGoogleCalendar(true);
     toggleGoogleDrive(!!args.extensionAPI.settings.get("drive-enabled"));
+
+    // Expose Google API functions
+    if (!window.roamjs.extension.google) {
+      window.roamjs.extension.google = {};
+    }
+    window.roamjs.extension.google.fetchGoogleContacts = fetchGoogleContacts;
+
     return {
       unload: () => {
         toggleGoogleCalendar(false);
         toggleGoogleDrive(false);
+        delete window.roamjs.extension.google.fetchGoogleContacts;
       },
     };
   },
 );
+
+// Add type declarations
+declare global {
+  interface Window {
+    roamjs: {
+      extension: {
+        google: {
+          fetchGoogleCalendar?: (options: any) => Promise<any>;
+          fetchGoogleContacts?: () => Promise<FormattedContact[]>;
```

Comment:
@mdroidian let's follow up this PR to add it to where the rest are declared in roamjs-components, then delete from here